Evergreen/Apps/Get-doublecmd.ps1

```diff
@@ -0,0 +1,27 @@
+Function Get-doublecmd {
```

Comment:
@g3rhard could you please rename the functions and file names from `doublecmd` to `DoubleCommander`.

Reply:
Of course, not a problem :)